.github/workflows/database.yml

```diff
@@ -0,0 +1,42 @@
+name: Database CI
```

Comment:
Ok I give you an annotated description of how Github Actions get set up:

Starts with a `.yml` file being placed into the `.github/workflows` folder. This file defines a "workflow", which Github will interpret